File: javaScript/problem1.js
//find multiples of 3 and 5!
function sumOfMultiples(max_num) {
  var sum = 0;
  var multiples = [];
  for (var i = 0, len = max_num; i < len; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      multiples.push(i);
    }
  }
  for (var i in multiples) {
    sum += multiples[i]
  }
  return sum;
}
console.log(sumOfMultiples(1000));

File: rubyChallenge/problem1.rb
# find all the multipes of 3 and 5 given an number, then output the sum.

def sum_of_multiples integer
  sum = 0;
  multiples_of_three_and_five = []
  
  integer.times do |num|
    multiples_of_three_and_five.push(num) if num % 3 == 0 || num % 5 == 0
  end

  sum = multiples_of_three_and_five.reduce(:+)
end
p sum_of_multiples 1000
